pkitesting/src/test/java/io/netty/pkitesting/CertificateBuilderTest.java

```diff
@@ -98,6 +99,7 @@ void createKeyPairOfEveryKeyType(Algorithm algorithm) throws Exception {
         assertNotNull(keyPair.getPublic());
     }
 
+    @Disabled
```

Comment:
@chrisvest can you investigate in this one ? We might want to merge in the meantime and do a followup ... WDYT ?